lib/core-server/src/utils/StoryIndexGenerator.ts

```diff
@@ -88,7 +88,7 @@ export class StoryIndexGenerator {
       const importPath = slash(normalizeStoryPath(relativePath));
       const makeTitle = (userTitle?: string) => {
         return userOrAutoTitleFromSpecifier(importPath, specifier, userTitle);
-      }
+      };
```

Comment:
Seems to be 3 or 4 random (prettier?) changes here, what's going on?

Reply:
Yeah, seems to me someone committed code without running prettier. That semicolon was definitely missing. 🤷